### DISCUSSION TITLE
Is there docs/plans introduction for serverless related work?

### DISCUSSION THREAD

**Discussion by feilengcui008** [databendlabs/databend#4744] [Q&A] [answered]
I've noticed the `serverless` doc change from this PR：https://github.com/datafuselabs/databend/commit/60e719df8a3b6e5681627ea41f6760c89f5e6cb5, wondering is there any introduction on the serverless part of databend?

**Comment by ZhiHanZ** [comment, marked as answer]
Hi @feilengcui008 , serverless deployment is under our roadmap and under active development, we would introduce it to public once it become stable
